tests/insert_license_test.py

```diff
@@ -99,16 +105,23 @@ def test_insert_license(license_file_path,
 
 
 @pytest.mark.parametrize(
-    ('license_file_path', 'src_file_path', 'comment_prefix'),
-    map(lambda a: a[:1] + a[1], product(  # combine license files with other args
-        ('LICENSE_with_trailing_newline.txt', 'LICENSE_without_trailing_newline.txt'),
-        (
-            ('module_with_license.groovy', '//'),
-            ('module_with_license_and_numbers.py', '#'),
-            ('module_with_year_range_in_license.py', '#'),
-            ('module_with_spaced_year_range_in_license.py', '#'),
+    ("license_file_path", "src_file_path", "comment_prefix"),
+    map(
+        lambda a: a[:1] + a[1],
+        product(  # combine license files with other args
+            (
+                "LICENSE_with_trailing_newline.txt",
+                "LICENSE_without_trailing_newline.txt",
+                "LICENSE_with_year_range_and_trailing_newline.txt",
+            ),
+            (
+                ("module_with_license.groovy", "//"),
+                ("module_with_license_and_numbers.py", "#"),
+                ("module_with_year_range_in_license.py", "#"),
+                ("module_with_spaced_year_range_in_license.py", "#"),
+            ),
         ),
-    )),
+    ),
 )
 def test_insert_license_current_year_already_there(license_file_path,
```

Comment:
It's nice to have a dedicated test function for `--use-current-year` usages 😊

However, this function does not validate the content of the file post-insertion,
as `test_insert_license()` or `test_fuzzy_match_license()`.
Could you please add a `new_src_file_expected` parameter to this function?

Reply:
If the current year is already there we don't expect any modification. I added a check to verify the contents of the file are identical before and after the `insert_license()` call.

Reply:
Perfect